MaterialsLib/Adsorption/Density100MPa.cpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "Density100MPa.h"
+
+namespace
+{
+
+// NaX_HighP_polyfrac_CC.pickle
+// date extracted 2015-06-23 15:38:35 file mtime 2015-06-23 15:19:57
+const double c[] = {
+    0.3490302932983226,        /* a0 */
+    -0.0014061345691831226,    /* a1 */
+    -0.0007399303393402753,    /* a2 */
+    5.129318840267485e-09,    /* a3 */
+    5.243619689772646e-07,    /* a4 */
+    6.347011955956523e-10,    /* a5 */
+    -9.919599580166727e-11    /* a6 */
+};
+
+}
+
+namespace Adsorption
+{
+
+double Density100MPa::get_adsorbate_density(const double T_Ads) const
+{
+    return -0.0013*T_Ads*T_Ads + 0.3529*T_Ads + 1049.2;
+}
+
+
+//Thermal expansivity model for water found in the works of Hauer
```

Comment:
I'd like to keep it as it is for now, since there is a ton of similar issues in the adsorption code.